config/optional/seckit.settings.yml

```diff
@@ -0,0 +1,54 @@
+seckit_xss:
+  csp:
+    checkbox: true
+    report-only: false
+    default-src: '''self'''
+    script-src: '''self'' ''unsafe-inline'' svc.webspellchecker.net'
+    object-src: ''
+    img-src: '''self'' data: reference.vic.gov.au *.reference.vic.gov.au content.reference.vic.gov.au *.content.reference.vic.gov.au *.amazee.io tracking.monsido.com *.google-analytics.com *.g.doubleclick.net www.google.com *.google.com *.google.com.au api.mapbox.com svc.webspellchecker.net'
```

Comment:
I think instead of *`.reference.vic.gov.au` or `content.reference.vic.gov.au` we need to make more generalised. So that in a new project install it does not add the reference site reference. So may be a wild card value like this - *.vic.gov.au 
After that if they want to put site specific changes, they can add it manually for that project only